webview/src/stories/Plots.stories.tsx

```diff
@@ -37,7 +37,7 @@ const smallCustomPlotsFixture = {
 }
 
 const manyCustomPlots = (length: number) =>
-  Array.from({ length }, () => customPlotsFixture.plots[2]).map((plot, i) => {
+  Array.from({ length }, () => customPlotsFixture.plots[1]).map((plot, i) => {
```

Comment:
[F] `plots[2]` no longer existing bricked all of the plot stories. Hence the reason that 25 new baselines now need to get accepted.